Documentation updates

micahellison opened the pull request:
Goals for this pr:

- [X] Apply changes from reviews of previous doc PRs - #980, #981, #982
- [x] Ensure that documentation is completely aligned with `jrnl --help` now that #1027 is merged

### Checklist

- [x] I have read the [contributing doc](https://github.com/jrnl-org/jrnl/blob/develop/CONTRIBUTING.md).
- [x] I have included a link to the relevant issue number.
- [x] I have tested this code locally.
- [x] I have checked to ensure there aren't other open [pull requests](../pulls)
  for the same issue.
- [ ] I have written new tests for these changes, as needed.
- [x] All tests pass.
<!--
NOTE: Your PR may not be reviewed if there are any failing tests. You can run
tests locally with `make test` (see the contributing doc if you need help with
`make`), or use our automated tests after you submit your PR.
-->

micahellison committed:
Applying doc changes based on reviews of past several documentation PRs

wren commented on docs/overview.md:
```diff
@@ -51,8 +48,8 @@ be [exported](./export.md) in a variety of formats.
 
-`jrnl` is compatible with most operating systems. Pre-compiled binaries are
-available through several distribution channels, and you can build from source.
+`jrnl` is compatible with most operating systems. You can download it using one
```
Should this link to the installation page?

micahellison replied:
Yes, definitely!

wren committed:
Update docs

Clean up encryption docs
Clean up security docs
Delete export.md
Make new formats.md and add to sidebar. Also add all of the built-in
formats, and examples for each.
Update mkdocs config for new files

micahellison committed:
Fix broken docs links

micahellison committed:
Correct incomplete sentences and markdown formatting issues

micahellison committed:
Make overview a little more concise

micahellison committed:
Update some command line arguments to latest version and make it a bit more concise

micahellison committed:
Clean up unneeded TOML modifications and other scaffolding not needed for 3.9

micahellison committed:
Revert "Clean up unneeded TOML modifications and other scaffolding not needed for 3.9"

This reverts commit 13b4266ed1e7150ef5a2235a2f816333692b71f2.

micahellison committed:
Specify that brew is also the easiest way to install jrnl on Linux

wren commented on docs/import-and-export.md:
```diff
@@ -0,0 +1,28 @@
+## Export to files
+
+You can specify the output file of your exported journal using the
+`-o` argument
```
This isn't true anymore. `-o` will still work, but only for backward compatibility. They should use `--file` like in `formats.md`.

wren replied:
Actually, do we still need this file? Maybe `formats.md` is enough to cover this?

wren commented on docs/security.md:
```diff
@@ -1,65 +1,80 @@
 ### zsh
 
-Disable history logging by adding this to your `zshrc`:
+You can disable history logging for jnrl by adding this to your `~/.zshrc`
```
```suggestion
You can disable history logging for jrnl by adding this to your `~/.zshrc`
```

wren commented on docs/encryption.md:
```diff
@@ -52,7 +35,7 @@ encrypted file untouched and create a new plain text file next to it.
 ## Storing Passwords in Your Keychain
 
-There is no method to recover or reset your `jrnl` password. If you lose it,
+You can't recover or reset your `jrnl` password. If you lose it,
```
Maybe "nobody can" could work here? It's also fine as-is.

wren commented on docs/recipes.md:
```diff
@@ -113,7 +113,7 @@ logged as a new entry in the journal you specified in the original argument.
 !!! tip 
     To read your journal entry or to verify the entry saved, you can use this 
-    command: `jrnl -n 1` (Check out [Import and Export](./export.md) for more export options).
+    command: `jrnl -n 1` (Check out [Formats](./formats.md) for more export options).
```
```suggestion
    command: `jrnl -n 1` (Check out [Formats](./formats.md) for more options).
```

wren commented on mkdocs.yml:
```diff
@@ -19,6 +19,7 @@ nav:
   - Privacy and Security: security.md
-  - Import and Export: export.md
+  - Formats: formats.md
+  - Import and Export: import-and-export.md
```
If we remove the export file above, we should take it out of here, too.

wren reviewed (changes requested):
This is looking fantastic. Just a few minor nitpicks, and I think we're good to go.

micahellison committed:
Update docs/security.md

Co-authored-by: Jonathan Wren <jonathan@nowandwren.com>

micahellison committed:
Update docs/recipes.md

Co-authored-by: Jonathan Wren <jonathan@nowandwren.com>

micahellison committed:
Doc updates:

- Remove import/export page, fold it into formats
- Rename security to privacy-and-security.md to avoid conflation w/ github security issues
- Various small cleanup and edits from PR review

wren reviewed (approved):
🤠
